packages/geofire-common/src/index.ts

```diff
@@ -29,6 +29,10 @@ export const E2 = 0.00669447819799;
 // Cutoff for rounding errors on double calculations
 export const EPSILON = 1e-12;
 
+export type geopoint = [number, number];
+export type geohash = string;
```

Comment:
Hmm I don't think aliasing `string` is worth it here since it doesn't tell you anything about the string.

Definitely having `GeoHashRange = [string, string]` is good but if we start looking for `geohash` in our input types rather than `string` then users will probably have to cast to make it work since they're probably getting the hashes from RTDB or Firestore.

Reply:
I have a lower priority task to set a maximum length for the Geohash, just need to figure out what it is and how to do that. Would that make the case for introducing a type alias stronger? I really want to introduce these types now, and then possibly tighten them going forward (like also setting valid value ranges on Geopoint elements).